src/UI/Component/Chart/Speedo/Factory.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/* Copyright (c) 2017 Ralph Dittrich <dittrich@qualitus.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Chart\Speedo;
+
+/**
+ * Interface Factory
+ * @package ILIAS\UI\Component\Chart\Speedo
+ */
+interface Factory {
+
+    /**
+     * ---
+     * description:
+     *   purpose: The standard speedo is used if the speedo is to be rendered in it's original size.
+     *
+     * rules:
+     *   composition:
+     *     1: Speedos MUST contain a goal value. It MUST be numeric and MAY be a score equal to 100%.
+     *     2: Speedos MUST contain a score. It MUST be numeric and MAY be equal to 0 or a equal to a value between 0% and 100% .

```

Comment:
MAY be equal to 0 or a equal to a value between 0% and 100% .
I think it would be clearer to state:
It MUST be a numeric value between 0 and goal (or maximum).